media/stylus/components/compat-tables/bc-table.styl

```diff
@@ -0,0 +1,308 @@
+    border-left: 0;
+    text-align: left;
+}
+
+
+/* td
+----------------------------------------------------------------------  */
+
+/* empty td in thead */
+.bc-table thead td {
+    border-top-color: transparent;
+    border-right: 4px solid $bc-color-border;
+    border-left: 0;
+}
+
+/* make outline more visible */
+.bc-table td:focus {
+
+    z-index: 1;
+}
+
+.bc-table td[tabindex] {
+    cursor: pointer;
+}
+
+.no-js .bc-table tbody td,
+.bc-table tbody td[tabindex] {
+    padding-top: 6px;
+    padding-bottom: 12px;
+}
+
+
+/* code
+----------------------------------------------------------------------  */
+.bc-table tbody th code {
+    white-space: nowrap;
+}
+
+
+/*
+tablet display
+- declarataions are mixins
+- mixins are included in media queries below
+----------------------------------------------------------------------  */
+
+bc-table-tablet-display-block() {
+
+    .bc-table thead,
+    .bc-table tbody,
+    .bc-table tr {
+        width: 100%;
+        display: block;
+        clear: left;
+    }
+
+    .bc-table th,
+    .bc-table td {
+        display: block;
+        float: left;
+    }
+
+    .bc-table tbody th + td {
+        clear: left;
+    }
+
+    .bc-table thead td {
+        display: none;
+    }
+
+    .bc-table tbody th {
+        width: 100%;
+        max-width: 100%;
+        border-right: 0;
+        padding-top: 10px;
+        padding-bottom: 0;
+    }
+}
+
+bc-table-tablet-display-flex() {
+    /*
+    Flexbox for those lucky few who can handle it
+    - we only needs this to make equal height columns in the tablet display view
+    - ie is not among those that can handle this flexbox code,
+      but it doesn't supoort @support so we need to include that in the MQ
+    - all browsers which support @support also use unprefixed syntax
+    ----------------------------------------------------------------------  */
+
+    .bc-table,
+    .bc-table thead,
+    .bc-table tbody,
+    .bc-table tr,
+    .bc-table th {
+        display: flex;
+    }
+
+    .bc-table,
+    .bc-table thead,
+    .bc-table tbody {
+        flex-direction: column;
+    }
+
+    .bc-table tr {
+        flex-wrap: wrap;
+        align-items: stretch;
+    }
+
+    .bc-table th,
+    .bc-table td {
+        flex: 1 1 0;
+        justify-content: center;
+    }
+
+    for $num in (2..8) { /* 1 is default for both colspan and flex */
+        .bc-table th[colspan={'"' + $num + '"'}] {
+            flex: $num $num 0;
+        }
+    }
+
+    .bc-table tbody th {
+        display: block;
+        flex: 1 1 100%;
+        justify-content: flex-start;
+    }
+
+    .bc-table td {
+        display: block; /* if we leave it table-cell it doesn't act like a flex-item */
+    }
+
+    .no-js .bc-table tbody td,
+    .bc-table tbody td[tabindex] {
+        /* flex is top aligned not center aligned so keep spacing at top same as no icons */
+        padding-top: 5px;
+    }
+}
+
+@media all and (min-width: 769px) {
```

Comment:
Interesting! I notice this is one number greater than the value used in `$media-query-mobile`. For the sake of maintainability, maybe it would make sense to create a new variable called `$media-query-mobile-max` and use it in the definitions of both `$media-query-mobile` and this media query?